package.json

```diff
@@ -37,12 +37,13 @@
     "babel-cli": "6.7.7",
     "babel-loader": "6.2.4",
     "babel-plugin-transform-object-assign": "6.8.0",
-    "babel-preset-es2015": "6.6.0",
+    "lodash-webpack-plugin": "^0.11.4",
+    "babel-preset-env": "^1.6.1",
     "cross-env": "1.0.7",
     "executive": "1.1.5",
     "jsdoc": "3.4.0",
     "nightwatch": "0.8.18",
-    "webpack": "1.13.0"
+    "webpack": "3.0.0"
```

Comment:
Why not put it to 3.6.0 or up, your code won't break but normally there are quite a few bugfixes above the base version.